src/ziggurat/config.clj

```diff
@@ -65,6 +65,8 @@
 (defn config-from-env [config-file]
   (clonfig/read-config (edn-config config-file)))
 
+(declare config)
```

Comment:
can we leave it? since it does not affect the behavior of the code. I think its a good practice to declare some vars/symbols first before using it....